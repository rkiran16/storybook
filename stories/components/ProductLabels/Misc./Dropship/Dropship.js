import { html } from "lit-html";

export const Dropship = ({ isDropship = true }) => {
  if (isDropship) {
    return html`
      <span class="position-relative me-2 d-flex align-items-center">
        <span class="m-2 material-icons">check</span>
        <strong class="sub-title-3 fw-bold">DROP SHIP</strong>
      </span>
    `;
  } else {
    return html`
      <span class="position-relative me-2 d-flex align-items-center">
        <span class="m-2 material-icons">clear</span>
        <strong class="sub-title-3 fw-bold">DROP SHIP</strong>
      </span>
    `;
  }
};
