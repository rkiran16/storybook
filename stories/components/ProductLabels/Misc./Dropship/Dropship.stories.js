import { Dropship } from "./Dropship";

export default {
  title: "Components/ProductLabels/Misc/Dropship",
  parameters: {
    layout: "centered",
    zeplinLink:
      "https://app.zeplin.io/project/62902a463701ffb0e532df2b/screen/63053ee9c9b39414a05fab9b",
  },
};

const DropshipTemplate = (args) => Dropship(args);

export const Standard = DropshipTemplate.bind({});
Standard.args = {
  isDropship: true,
};
