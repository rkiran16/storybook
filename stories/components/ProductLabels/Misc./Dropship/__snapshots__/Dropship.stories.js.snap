// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Components/ProductLabels/Misc/Dropship Standard 1`] = `
Object {
  "_$litType$": 1,
  "strings": Array [
    "
      <span class=\\"position-relative me-2 d-flex align-items-center\\">
        <span class=\\"m-2 material-icons\\">check</span>
        <strong class=\\"sub-title-3 fw-bold\\">DROP SHIP</strong>
      </span>
    ",
  ],
  "values": Array [],
}
`;
